types/sweet-pubsub/index.d.ts

```diff
@@ -0,0 +1,53 @@
+// Type definitions for sweet-pubsub 1.1
```

Comment:
I think your definition is wrong because the package uses `module.exports` and therefore you need to create typing based on this pattern

```typescript
declare interface SweetPubsub {
    on():void;
    off():void;
    ...
}

declare const sweetPubsub: SweetPubsub;

export = sweetPubsub;
```

and you could use in the tests `import sweetPubsub = require('sweet-pubsub');` instead of  `import * as pubsub from 'sweet-pubsub';`

Reply:
sorry for that, it's fine

Reply:
Hi, thanks, i didn't notice that yes, it is a UMD library, i have updated the definitions as per the suggestions